### PR TITLE
Multiple themes management

### PR DESCRIPTION
**Pull request by fsansalvadore**
## What kind of change does this PR introduce?

- Add multi-theme support to `docs`, `www` and `studio`.
- Update all light/dark mode detection logic on:
  - images
  - tailwind `dark:` mode selectors
  - styling
- Add styling tokens:
  - `brand-button`
  - `background-button-DEFAULT`
  - `background-button-hover`
- Update remaining missing tokens

### CODE REVIEW

File: packages/ui/src/components/Toast/Toast.module.css [hunk 1]
```diff
@@ -2,8 +2,7 @@
   @apply max-w-xs w-full shadow-lg rounded-md pointer-events-auto overflow-hidden py-4 px-2;
 
   @apply border border-solid;
-  @apply bg-white border-gray-300;
-  @apply dark:bg-dark-700 dark:border-gray-400;
+  @apply bg-syrface-100 border-default;
```
Comment (saltcod) [inline packages/ui/src/components/Toast/Toast.module.css:5]: typo?

**Comment by saltcod** [comment]
screenshot-2023-11-11-at-08 41 40

the notification number isn't visible — not sure if related to this PR tbh

**Comment by saltcod** [comment]
screenshot-2023-11-11-at-08 47 47

**Comment by saltcod** [comment]
i think this is the light warning colour?
screenshot-2023-11-11-at-08 49 28

**Comment by saltcod** [comment]
icon fill not right here
screenshot-2023-11-11-at-08 50 25

**Comment by saltcod** [comment]
screenshot-2023-11-11-at-08 52 13

**Comment by saltcod** [comment]
screenshot-2023-11-11-at-08 56 11

**Comment by saltcod** [comment]
screenshot-2023-11-11-at-08 57 20

File: packages/ui/src/components/Toast/Toast.module.css (hunk 1 above)
Comment (fsansalvadore) [reply packages/ui/src/components/Toast/Toast.module.css]: Good catch, fixed 👍

**Review by MildTomato — APPROVED** [review]

**Review by saltcod — APPROVED** [review]

**Pull request merged**